### DISCUSSION TITLE
why no "vendor" in device list possible?

### DISCUSSION THREAD

**Discussion by kamil-malinski** [netalertx/NetAlertX#154] [Ideas]
Hi!

Now that we can customize the colls in the device list I wonder why we can not add the "vendor" to the list?

Cheers

Kamil

**Comment by jokob-sk** [comment]
Hi,

Should be part of the next release: https://github.com/jokob-sk/Pi.Alert/commit/14cb0f46f3d43026610d69a6a278d2b54566b450

j
